src/main/scala/com/wavesplatform/metrics/Instrumented.scala

```diff
@@ -1,78 +1,94 @@
 package com.wavesplatform.metrics
 
 import com.wavesplatform.utils.ScorexLogging
-import kamon.metric.{Counter, Histogram}
-
-trait Instrumented {
-  self: ScorexLogging =>
+import kamon.metric.{Counter, Histogram, MeasurementUnit}
 
+trait Instrumented { self: ScorexLogging =>
```

Comment:
I suggest removing all methods duplicating `Timer` functionality and deprecating all `measure*Log` methods.

Reply:
Ok